index.js

```diff
@@ -21,24 +16,24 @@ function buildTargets(options) {
   return assign({}, defaultTargets, options.additionalTargets);
 }
 
-module.exports = function buildAirbnbPreset(context, options) {
+module.exports = function buildAirbnbPreset(api, options) {
```

Comment:
Might be worth considering the `declare` helper that Babel uses. https://github.com/babel/babel/blob/master/packages/babel-preset-flow/src/index.js#L4

I'd also include `api.assertVersion(7);` so users get a nice error. That also guarantees that `options` is an object, so all the truthy checks on `options` could be removed.